csaf_2.0/json_schema/csaf_json_schema.json

```diff
@@ -974,9 +974,9 @@
                 "type": "string",
                 "pattern": "^CWE-[1-9]\\d{0,5}$",
                 "examples": [
-                  "CWE-79",
                   "CWE-22",
-                  "CWE-352"
+                  "CWE-352",
+                  "CWE-79"
```

Comment:
Shouldn't that be:
```suggestion
                  "CWE-22",
                  "CWE-79",
                  "CWE-352"
```

Reply:
Well, we order lexically. The prefix makes the 7 in the CWE-79 appear after the CWE-352 due to the leading 3 of the latter number part. Personally I think CWE numbers have long fallen victim to the _initial plan_ Syndrome, where at start you think you know the terrain and set numbers, and then life goes on and all the balancing is gone as well as some of these well meant advice documents that require deletion as time goes by.

Reply:
> Well, we order lexically. The prefix makes the 7 in the CWE-79 appear after the CWE-352 due to the leading 3 of the latter number part.

Makes totally sense. I agree. I withdraw the comment ;-)